charts/sessionspaces/charts/sessionspaces/values.yaml

```diff
@@ -0,0 +1,47 @@
+nameOverride: ""
+fullnameOverride: ""
+
+image:
+  repository: ghcr.io/diamondlightsource/workflows
+  pullPolicy: Always
+  tag: ""
+
+imagePullSecrets: []
+
+logLevel: Warn
+otelCollectorUrl: ""
+
+database:
+  host: ""
+  user: ""
+  password:
+    secretName: ""
+    secretKey: ""
+
+opa:
+  url: ""
+
+replicaCount: 1
+
```

Comment:
Do we support replication? Isn't the controller stateful? Should we aim for HA? If so, how?